api/src/routes/index.ts

```diff
@@ -1,10 +1,10 @@
 import express = require("express");
 const router = express.Router();
 
-router.use(function timeLog(req, res, next) {
-    console.log("Time: ", Date.now());
-    next();
-});
+// router.use(function timeLog(req, res, next) {
```

Comment:
Should we just delete this? Not sure what it has to do with the rest of this PR.